package.json

```diff
@@ -1,15 +1,16 @@
 {
   "name": "@magnet.me/logger-js",
-  "version": "1.0.3",
+  "version": "1.0.4",
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
-  "types": "dist/index.d.ts",
+  "types": "dist/esm/index.d.ts",
   "repository": "https://github.com/Magnetme/logger-js",
   "author": "Magnet.me",
   "license": "MIT",
   "private": false,
   "scripts": {
     "build": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json",
+    "rebuild": "rm -rf dist && yarn run build",
```

Comment:
`rebuild` is a reserved command in `yarn`, so I'm not sure if that's a good idea. I'd just go with a separate `clean`.

Reply:
renamed it.